### PR TITLE
cn: more tests to increase the coverage of cn pkg

### PR DESCRIPTION
**Pull request by ehnuje**
## Proposed changes

- added more tests to increase the coverage of cn package
- call `handleMsg` instead of specific message handling functions to increase the coverage of `handleMsg`.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by ehnuje** [comment]
@kjhman21 Below error is something expected?

```
...
go: finding go.uber.org/zap v1.10.0
go: finding go.uber.org/atomic v1.4.0
go: finding go.uber.org/multierr v1.2.0
go: finding github.com/pbnjay/memory v0.0.0-20190104145345-974d429e7ae4
go: finding github.com/urfave/cli v1.20.0
this is not RC version
```

**Comment by whoisxx** [comment]
@ehnuje circleci build job fail when it assign variable on BASH_ENV after checking RC version. I rerun the same job, it works now.

**Comment by ehnuje** [comment]
@nogang @KimKyungup Dear reviewers, could you please review this change?

File: node/cn/api_tracer_test.go [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+	mockCtrl, mockEngine, mockBlockChain, _ := newMocks(t)
+	mockMiner := mocks3.NewMockMiner(mockCtrl)
+	api := NewPrivateDebugAPI(nil, &CN{miner: mockMiner, blockchain: mockBlockChain, engine: mockEngine})
+	return mockCtrl, api, mockEngine, mockBlockChain, mockMiner
+}
+
+func TestPrivateDebugAPI_TraceChain(t *testing.T) {
+	endBlockNumber := rpc.BlockNumber(123)
+	block := newBlock(123)
+	{
```
Comment (KimKyungup) [inline node/cn/api_tracer_test.go]: These all error cases covers only `from = nil`.
Why don't you add `from != nil && to == nil` case?

Comment (ehnuje) [reply node/cn/api_tracer_test.go]: I thought I did it =( I will modify the test to cover that case. Thanks!

**Comment by ehnuje** [comment]
@KimKyungup `err` was a static variable used for expected error. I changed its name to `expectedErr`. Could you please review the change again?

**Comment by ehnuje** [comment]
@KimKyungup @jeonghonah @nogang @kjhman21 Dear reviewers, could you please review this change?

**Pull request merged**